app/models/user.rb

```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
+end
```

Comment:
Greetings, Are you agree to add these associations.
  has_many :favorites
  has_many :computers, through: :favorites